CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## ASPEN development version
 
+- Fix Diffatac error (#100, @kopardev): Adds a defensive check to prevent invalid 'row.names' length error when up_roi or down_roi are empty (due to strict FC/FDR thresholds in DiffATAC)
+- Minor refactoring to accomodate moving to ccbr_tools v0.4+ 
```

Comment:
The changelog should have references to the PR that contains the changes (not the issue number)

```suggestion
- Fix Diffatac error (#101, @kopardev)
  - Adds a defensive check to prevent invalid 'row.names' length error when up_roi or down_roi are empty (due to strict FC/FDR thresholds in DiffATAC)
- Minor refactoring to accomodate moving to ccbr_tools >= v0.4 (#101, @kopardev)
```